job_service/adapter/maintenance_db.py

```diff
@@ -27,33 +26,39 @@ def set_status(status_request: MaintenanceStatusRequest):
     try:
         document = {
             "msg": status_request.msg,
-            "pause": status_request.pause,
+            "paused": status_request.paused,
             "timestamp": str(datetime.now()),
         }
         doc_id = maintenance.insert_one(document)
+        return {"_id": doc_id, **document}
```

Comment:
we actually usually avoid returning the _id field in the other queries. I think that is reasonable. Maybe we should just return document here 👍🏻